cypress/fixtures/fireboltCalls/account.json

```diff
@@ -1,88 +1 @@
-{
-    "FETCH_ACCOUNT_ID":{
-        "method": "account.id"
-    },
-    "FETCH_ACCOUNT_UID":{
-        "method": "account.uid"
-    },
-    "GET_ACCOUNT_ID":{
-        "method": "account.id",
-        "expected": "error"
-    },
-    "ACCOUNT_ID": {
-		"method": "account.id",
-		"validationJsonPath": "result",
```

Comment:
v1 objects should not be deleted. we should have v1 variables for some time

Reply:
retained json objects

Reply:
did we confirm if we keep both json and js objects, the JSON objects are taken instead of js objects ?